ScoutSuite/output/data/html/partials/aws/services.cloudtrail.regions.id.trails.html

```diff
@@ -26,6 +28,7 @@ <h4>Information</h4>
         </li>
         <li class="list-group-item-text">Log File Validation Enabled: <span id="cloudtrail.regions.{{region}}.trails.{{@key}}.LogFileValidationDisabled">{{LogFileValidationEnabled}}</span></li>
         <li class="list-group-item-text">Destination S3 bucket name: <a href="javascript:showObject('services.s3.buckets.{{bucket_id}}')">{{get_value_at 'services.s3.buckets' bucket_id 'name'}}</a>/{{S3KeyPrefix}}</li>
+        <li class="list-group-item-text">KMS key: <span id="cloudtrail.regions.{{region}}.trails.{{@key}}.KmsKeyId">{{#if KmsKeyId}} true {{else}} false {{/if}}</span></li>
```

Comment:
I believe the `id_suffix` is wrong here too:

```suggestion
        <li class="list-group-item-text">KMS key: <span id="cloudtrail.regions.{{region}}.trails.{{@key}}.cloudtrail-kms-key-unused">{{#if KmsKeyId}} true {{else}} false {{/if}}</span></li>
```